GooglePlayInstant/Editor/QuickDeploy/AssetBundleBrowserClient.cs

```diff
@@ -124,11 +124,15 @@ public static void DisplayAssetBundleBrowser()
         {
             if (!AssetBundleBrowserIsPresent())
             {
-                Debug.LogError("Cannot detect Unity Asset Bundle Browser");
-                return;
+                throw new Exception("Cannot detect Unity Asset Bundle Browser. The browser was not found.");
```

Comment:
Failed to detect the Unity AssetBundle Browser.

Reply:
Resolved.